Protos/V1/tasks_common.proto

```diff
@@ -184,6 +184,79 @@ message CancelTasksResponse {
   repeated TaskSummary tasks = 1;
 }
 
+/**
+ * Request to count tasks.
+ */
+message CountTasksRequest {
+  /**
+   * Represents every availaible field to use in order to group tasks.
+   */
+  enum GroupByField {
+    GROUP_BY_FIELD_UNSPECIFIED = 0; /** Unspecified. */
+    GROUP_BY_FIELD_STATUS = 1; /** The task status. */
+    GROUP_BY_FIELD_SESSION_ID = 2; /** The session ID. */
+    GROUP_BY_FIELD_APPLICATION_NAME = 3; /** The application name. */
```

Comment:
Before that, we must have a `GROUP_BY_FIELD_PARTITION`.

Reply:
@esoubiran-aneo

Reply:
à ajouter avant tous les autres